### ISSUE TITLE
Org switch does not work in 8.x

### ISSUE THREAD

**Issue by sanchomuzax** [digiapulssi/grafana-organisations-panel#16]
The plugin seems working in 8.x in general, but button links are broken.
If I changed the plugin dimensions as an editor, the buttons will appear and appear for a while, then disappear again with a new session.

**Comment by sanchomuzax** [comment]
I am attaching two screenshots. In the first case, you can see that the DIV section is empty, the buttons are not loaded. The second screen was made after I moved the edge of the container with the mouse, which caused the buttons to load as well.

After loading:
![image](https://user-images.githubusercontent.com/2911588/186409376-f4b96190-090f-422a-b4bc-0532785f8ea0.png)

After mouse poke:
![image](https://user-images.githubusercontent.com/2911588/186409393-c7278a09-8b21-49c4-a29a-73676f4a24f7.png)

**Comment by timyllym** [comment]
Yep, it's not a biggie. The problem is that I haven't updated the panel for a while, and now to get it again approved by Grafana I'll need to port the panel to React (it is now Angular.js). So it needs a little effort to get the updated version to Grafana's plugin repository. But I'll try to make that as soon as possible.

**Comment by timyllym** [comment]
I have migrated this panel to React to support latest Grafana version. That is however a new panel this old panel is deprecated. The new panel can be found with id timomyl-organisations-panel https://grafana.com/grafana/plugins/timomyl-organisations-panel/ So closing this issue now.

**Comment by sanchomuzax** [comment]
Thank you! 🥇
